app/ui/legacy/src/main/java/com/fsck/k9/ui/account/AccountFallbackImageProvider.kt

```diff
@@ -5,14 +5,14 @@ import android.graphics.drawable.Drawable
 import androidx.core.content.ContextCompat
 import androidx.core.graphics.BlendModeColorFilterCompat
 import androidx.core.graphics.BlendModeCompat
-import com.fsck.k9.ui.R
+import app.k9mail.core.ui.legacy.designsystem.atom.icon.Icons
 
 /**
  * Provides a [Drawable] for the account using the account's color as background color.
  */
 class AccountFallbackImageProvider(private val context: Context) {
     fun getDrawable(color: Int): Drawable {
-        val drawable = ContextCompat.getDrawable(context, R.drawable.drawer_account_fallback)
+        val drawable = ContextCompat.getDrawable(context, Icons.Outlined.Person)
```

Comment:
The icon doesn't work well in a circular image view. It looks like there should be additional padding and/or the content should be moved up a bit.

For the drawable to work with Material Drawer's `ProfileDrawerItem` it seems like the width and height need to be at least 56dp.

When using `Icons.Outlined.Person` the account switcher looks like this:
![image](https://github.com/thunderbird/thunderbird-android/assets/218061/347cf83d-d6a8-4a7e-b2d5-71195c94e5ff)

I suggest we keep the old drawable and defer changing this to after we have a new design for the drawer.

Reply:
Not ideal, but I fixed it for now.